test/global_config_test.py

```diff
@@ -0,0 +1,59 @@
+"""Test global_config.py module"""
+from unittest import TestCase
+from os.path import expanduser
+
+from projector_installer.global_config import get_changelog_url, get_path_to_license, \
+    get_apps_dir, get_run_configs_dir, get_ssl_properties_file, get_download_cache_dir, \
+    get_ssl_dir, get_projector_server_dir
+
+
+class GlobalConfigTest(TestCase):
+    """Test global_config.py module"""
+
+    user = expanduser('~')
+
+    def test_get_changelog_url(self):
+        """The get_changelog_url method must return changelog url with the specified version"""
+        changelog_url = 'https://github.com/JetBrains/projector-installer' \
+                        '/blob/master/CHANGELOG.md#2020131'
+        self.assertEqual(get_changelog_url(ver='2020.13.1'), changelog_url)
+
+    def test_get_path_to_license(self):
```

Comment:
This test always fails if project is located in place other than ~/PycharmProjects/projector-installer/projector_installer.